robocop/checkers/community_rules/keywords.py

```diff
@@ -0,0 +1,89 @@
+from robot.api import Token
+from robot.utils.robottime import timestr_to_secs
+
+from robocop.checkers import VisitorChecker
+from robocop.rules import Rule, RuleParam, RuleSeverity
+from robocop.utils.misc import normalize_robot_name
+
+rules = {
+    "10001": Rule(
+        RuleParam(
+            name="max_time", default=0, converter=timestr_to_secs, desc="Maximum amount of time allowed in Sleep"
+        ),
+        rule_id="10001",
+        name="sleep-keyword-used",
+        msg="Sleep keyword with '{{ sleep_time }}' sleep time found",
```

Comment:
What do you think about using `duration time` instead of `sleep time`? (no strong opinion)

Reply:
I like it better, ``sleep time`` sounds quirky for me too